### PR TITLE
Shayan/ Rest of new design modification

### PR DESCRIPTION
**Pull request by shayan-binary-2**

### CODE REVIEW

File: sass/components/chart-mode.scss [hunk 1]
```diff
@@ -59,7 +59,7 @@
 }
 
 .smartcharts-mobile {
-    .sc-chart-mode .cq-dialog {
+    .sc-chart-mode .sc-dialog {
```
Comment (zaki-binary) [inline sass/components/chart-mode.scss:62]: @shayan-binary-2 , please note that in deriv.app there are instances where we are overriding the classNames with `cq` prefix. Just to give a heads up again.

Comment (shayan-binary-2) [reply sass/components/chart-mode.scss:62]: @zaki-binary yes exactly. after releasing the chart, I am going to integrate it with deriv.app and update any places which we use these overriding

File: app/test.jsx [hunk 2]
```diff
@@ -237,8 +243,6 @@ class App extends Component {
         <>
             <ChartTitle onChange={this.symbolChange} />
             {this.state.settings.historical ? <ChartHistory onChange={this.handleDateChange} /> : ''}
```
Comment (zaki-binary) [inline app/test.jsx:245]:
```suggestion
            {!!this.state.settings.historical && <ChartHistory onChange={this.handleDateChange} />}
```

File: app/test.jsx [hunk 3]
```diff
@@ -289,25 +293,52 @@ class App extends Component {
...
+            <StudyLegend portalNodeId="portal-node" />
+            <Views portalNodeId="portal-node" />
+            <DrawTools portalNodeId="portal-node" />
+            <Share portalNodeId="portal-node" />
+        </ToolbarWidget>
+    );
+
     onMessage = (e) => {
         this.notifier.notify(e);
     }
```
Comment (zaki-binary) [inline app/test.jsx:333]:
```suggestion
    onMessage = (e) => this.notifier.notify(e);
```

File: sass/components/dialog.scss [hunk 4]
```diff
@@ -14,12 +14,12 @@
         background: themed('DialogBodyBg');
     }
 
-    &.cq-dialog-enter-done, &.cq-dialog-enter-active {
+    &.sc-dialog-enter-done, &.sc-dialog-enter-active {
         opacity: 1;
         transform: translateY(0em);
```
Comment (zaki-binary) [inline sass/components/dialog.scss:19]:
```suggestion
        transform: none;
```

File: app/test.jsx (hunk 2 above)
Comment (shayan-binary-2) [reply app/test.jsx]: @zaki-binary modified

File: app/test.jsx (hunk 3 above)
Comment (shayan-binary-2) [reply app/test.jsx]: @zaki-binary modified

File: sass/components/dialog.scss (hunk 4 above)
Comment (shayan-binary-2) [reply sass/components/dialog.scss]: @zaki-binary modified

**Pull request merged**